lib/util/cli.js

```diff
@@ -37,24 +38,22 @@ const Cli = {
      * @returns {Promise.<void>} -
      */
     async addExtraCredential(properties, skipInteraction) {
-        if (!(await Util.checkProperties(properties))) {
-            // return null here to avoid seeing 2 error messages for the same issue
-            return null;
-        } else {
-            Util.logger.info('Found the following credentials in your config file:');
-            for (const cred in properties.credentials) {
-                if (Object.prototype.hasOwnProperty.call(properties.credentials, cred)) {
-                    Util.logger.info(` - ${cred}`);
-                }
-            }
-            Util.logger.info('\nPlease enter your new credentials');
-            if (skipInteraction && properties.credentials[skipInteraction.credentialName]) {
-                Util.logger.error(
-                    `Credential '${skipInteraction.credentialName}' already existing. If you tried updating please provide run 'mcdev init ${skipInteraction.credentialName}'`
-                );
+        // check existing properties before adding additional // todo CONFIRM if needed
+        await config.checkProperties(properties);
```

Comment:
is an error here handled higher up in the stack? does running this even make sense, considering you are running it inside of getProperties()?

Reply:
push